### PR TITLE
Ecommerce Backend initial version PR

### PR DESCRIPTION
**Pull request by shubham-ghuge**
Hey, Please review my code.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+app.use(cors())
+app.use(express.static('public'))
+initialiseDbConnection();
+
+app.use('/products', productRouter);
+
+app.get('/', (req, res) => {
+  res.json({ 'Hello From Farmers Grocery!': 'API' })
+});
+
```
Comment (prithviBytes) [inline app.js:20]: Hi Shubham,
Over here could you please add an invalid route handler?
Something like 404 route not found!

File: db/connect.db.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+require('dotenv').config();
+
+async function initialiseDbConnection() {
+  await mongoose.connect(
+    process.env['mongodb_uri'], { useUnifiedTopology: true, useNewUrlParser: true })
+    .then(console.log('Mongo DB connected'))
+    .catch(err => console.log("error in mongodb connection", err));
+}
```
Comment (prithviBytes) [inline db/connect.db.js]: Hey,
Over here you are usingboth async await and promises. Use one of them.
I would suggest you go for async await and surround it with try catch block.

```
try{
     await mongoose.connect()
}catch(err){
  do something
}

**Review by prithviBytes — CHANGES_REQUESTED** [review]
Hi Shubham,
The code looks good, I have requested 2 changes. Do ping me for further clarifications if needed.
Thanks,
Prithvi

**Comment by shubham-ghuge** [comment]
> Hi Shubham,
> The code looks good, I have requested 2 changes. Do ping me for further clarifications if needed.
> Thanks,
> Prithvi

Thank you for your time, Prithvi.
I have completed the changes.

**Review by prithviBytes — APPROVED** [review]
Hi Shubham,
Thanks for the changes.
I am approving the PR from my end.

**Pull request merged**